### PR TITLE
Adding tasks files

### PR DESCRIPTION
**Pull request by UsmanArshad-Cloud**

### CODE REVIEW

File: Binary_Operators.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+def compute_bitwise(a,b):
```
Comment (haxhmi1993) [inline Binary_Operators.py:1]: we need to write a program where we can return the set bits of a number
100= set bids is 1
1010 = set bids 2
we need to pass an integer number and we will return the set bits count of that number binary

File: Enums_and_Switches.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    PAK = "PAKISTAN"
+    IND = "INDIA"
+    AUS = "AUSTRALIA"
+    ENG = "ENGLAND"
+    BAN = "BANGLADESH"
+    NZ = "NEWZEALAND"
+
+
+value = input("Give the abbreviated country name:")
+match value:
```
Comment (haxhmi1993) [inline Enums_and_Switches.py:14]: create a function and pass value as enum so you don't have to compare with value

File: Group_By.py [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    {"Name": "Ahmed", "FatherName": "Rasheed", "Age": 28, "Location": "Gulberg"},
+    {"Name": "Zara", "FatherName": "Siddique", "Age": 31, "Location": "Defence"},
+    {"Name": "Hassan", "FatherName": "Malik", "Age": 26, "Location": "Gulberg"},
+    {"Name": "Fatima", "FatherName": "Rizvi", "Age": 29, "Location": "Shahdra"},
+    {"Name": "Bilal", "FatherName": "Azam", "Age": 24, "Location": "Model Town"},
+    {"Name": "Hina", "FatherName": "Ali", "Age": 33, "Location": "Defence"}
+]
+
+
+def GroupByFunc(list):
```
Comment (haxhmi1993) [inline Group_By.py:15]: we need to pass 2 parameters list and key ( key on which we want to group so the function is generic ).
We need to do same function using reduce function

File: Map_Task.py [hunk 4]
```diff
@@ -0,0 +1,8 @@
+def double_num(num):
+    return num*2
+
+array=[1,2,3,4,5]
+
+doubled_array=list(map(double_num,array))
```
Comment (haxhmi1993) [inline Map_Task.py:6]: create list of object with 2 keys {a:1,b:2} and add 1 more key c:3 using map function

File: Polymorphism_Python.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
+#Function Overloading
```
Comment (haxhmi1993) [inline Polymorphism_Python.py:1]: Polymorphism is not overloading of function, its overriding. So use classes

File: Power_Recrsion.py [hunk 6]
```diff
@@ -0,0 +1,27 @@
+def Power_Iterative(base,exponent):
```
Comment (haxhmi1993) [inline Power_Recrsion.py:1]: A function should be generic and should be production ready. If you use power function of math library, you will see that they are dealing with negative numbers too

File: Primary_Numbers.py [hunk 7]
```diff
@@ -0,0 +1,26 @@
+import math
+
+
+def compute_primary_number(n):
+    prime_count = 0
+    primary_list = []
+    current = 2
+    while True:
+        if prime_count == n:
```
Comment (haxhmi1993) [inline Primary_Numbers.py:9]: where we need if we can use the condition in while loop

Comment (haxhmi1993) [reply Primary_Numbers.py:9]: We need to write function with can print prime numbers between 1-100

File: Binary_Operators.py (hunk 1 above)
Comment (UsmanArshad-Cloud) [reply Binary_Operators.py:1]: I have pushed the solution

Comment (haxhmi1993) [reply Binary_Operators.py:1]: Did you push on the same branch?

**Pull request merged**